source/Projectile.cpp

```diff
@@ -71,18 +72,20 @@ Projectile::Projectile(const Projectile &parent, const Point &offset, const Angl
 	cachedTarget = TargetPtr().get();
 	double inaccuracy = weapon->Inaccuracy();
 	if(inaccuracy)
-	{
 		this->angle += Angle::Random(inaccuracy) - Angle::Random(inaccuracy);
-		if(!parent.weapon->Acceleration())
```

Comment:
Just realized why this only worked on weapons without acceleration; as soon as acceleration is introduced, the original calculations before #5515 didn't hold true because acceleration alters the velocity while in flight, but the old calculations used the weapon's velocity stat. #5515 changed to using the actual velocity, but this PR goes back to relying on the weapon stat, so we should follow what was done and only apply this to on-accelerating parent projectiles.

Reply:
Yeah that's an issue we're going to have to work around, I think.  I don't know how much we can infer from the currently available data though.   It'll have to be weighed against any slowdown it causes, but we really could use with another attribute in the data files on submuitions (technically it could apply to primary munitions but I don't haven't thought through if that makes sense yet) that tells us how much velocity to redirect as inaccuracy.

Reply:
I'm curious if we can extend support to all submunitions by doing the reverse of what is done: rather than store the initial speed of this projectile, store the final velocity of the parent. Any difference in magnitude--for projectiles that cannot turn--is then added speed from linear acceleration.

Alternately, if we update the projectile's `speed` at the end of its life (right before it would spawn submunitions), would that not solve the issue with discerning exactly the added velocity component from the submunition, even if the parent was able to accelerate or turn?